recipe/meta.yaml

```diff
@@ -15,11 +15,13 @@ build:
 
 requirements:
   build:
+    - libtool  # [unix]
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
     - m4
     - perl
     - make
+    - flex   # [build_platform != target_platform]
```

Comment:
Why is this needed in `build` in addition to `host`?

Reply:
I can't remember exactly, but bison configure was running the flex executable for something